src/backend/joanie/core/models/products.py

```diff
@@ -804,12 +849,23 @@ def clean(self):
                     error_dict["order_group"].append(
                         f"An active order group is required for product {product_title:s}."
                     )
-                else:
-                    nb_seats = self.order_group.nb_seats
-                    if 0 < nb_seats <= self.order_group.get_nb_binding_orders():
+                elif (
+                    self.order_group.is_enabled
```

Comment:
You must also update the logic on the line 848.

Furthermore the queryset line 846 will be problematic I think but it's trickier to fix.

So I suggest to fix in this PR the line 846 then about the line 848 I let you the choice to fix that right now in this PR or create an issue then address this point in a dedicated PR